src/views/TradingCompetition/index.tsx

```diff
@@ -70,15 +74,18 @@ const TradingCompetition = () => {
   const { t } = useTranslation()
   const { profile, isLoading } = useProfile()
   const { isDark, theme } = useTheme()
-  const tradingCompetitionContract = useTradingCompetitionContract()
-  const [currentPhase, setCurrentPhase] = useState(CompetitionPhases.CLAIM)
+  const tradingCompetitionContract = useTradingCompetitionContractV2(false)
+  const [currentPhase, setCurrentPhase] = useState(CompetitionPhases.REGISTRATION)
   const [registrationSuccessful, setRegistrationSuccessful] = useState(false)
   const [claimSuccessful, setClaimSuccessful] = useState(false)
-  const [userTradingInformation, setUserTradingInformation] = useState({
+  const [userTradingInformation, setUserTradingInformation] = useState<UserTradingInformationProps>({
     hasRegistered: false,
     hasUserClaimed: false,
     userRewardGroup: '0',
     userCakeRewards: '0',
+    userLazioCakeRewards: '0',
```

Comment:
should this be `userLazioRewards` instead of userLazio**Cake**Rewards?
Same with Porto and Santos